src/components/AddBook.jsx

```diff
@@ -0,0 +1,9 @@
+const AddBook = () => (
+  <form>
+    <h1>Create New Book</h1>
+    <input type="text" placeholder="Book title" />
+    <input type="text" placeholder="Author" />
+    <input type="submit" value="Add book" />
+  </form>
+);
+export default AddBook;
```

Comment:
- [ ] Nothing displays on your page at this moment. Kindly check your configuration and fix whenever there're issues.

### Page
![your-page](https://github.com/batoolfatima2135/book-store/assets/26597432/1c41a9ed-d868-427a-aa3a-368c6c57c213)

Reply:
Hey @Qoosim, 
It was not required to render all the components 😀 but still I have rendered it now on my homepage.